src/Core/Traits/PrivateDeployment.php

```diff
@@ -39,7 +39,7 @@ public function isPrivate()
      * Check if user can access deployment
      * @return boolean
      */
-    public function canAccessDeployment(User $user)
+    public function canAccessDeployment($user)
```

Comment:
removed type hint here because of an issue with the user I'm sending now but making a note  to reconsider since it'd be better just to send the correct user entity even if I have to cast it for the moment